src/Authentication/Authentication/custom/Find-MgGraphPermission.ps1

```diff
@@ -15,12 +15,18 @@ Find permissions for authorization against Microsoft Graph
 
 # Import-Module -Force .\Microsoft.Graph.Authentication.psd1
 
-function Find-MgGraphPermission($search) {
+function Find-MgGraphPermission([string]$search) {
```

Comment:
Need to add [[cmdletbinding]](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_functions_cmdletbindingattribute?view=powershell-7.1)  and use the `param` syntax for parameters, i.e.

```powershell
function Find-MgGrapPermission {
    [cmdletbinding()]
    param (
       [string] $search
    )
    # the rest of the code for the function.
}